src/index.ts

```diff
@@ -236,8 +236,6 @@ export {
 export type {
   ParseOptions,
   SourceLocation,
-  TokenKindEnum,
-  KindEnum,
```

Comment:
this is strange.
Problems with rebase?

Reply:
I think my editor was telling me these were deprecated, but I can try bringing them back.